client/cs_statefile.cpp

```diff
@@ -856,7 +856,8 @@ int CLIENT_STATE::write_state_file_if_needed() {
 
 // look for app_versions.xml file in project dir.
 // If find, get app versions from there,
-// and use "anonymous platform" mechanism for this project
```

Comment:
Apologies for being the grammar police, but shouldn't the line above say "If found" instead of "If find"?

Reply:
I think that 'If find' is comment-shorthand for 'If you find it'.